crates/aptos/src/node/mod.rs

```diff
@@ -1440,6 +1444,20 @@ impl CliCommand<()> for AnalyzeValidatorPerformance {
                     true,
                 );
             }
+            if print_max_tps {
+                for (num_blocks_for_max_tps, max_tps) in &epoch_stats.max_tps_per_block_interval {
+                    println!(
+                        "In {}epoch {}: during {} consecutive blocks, peak of {} TPS, ending on version: {}, {} txns over {}s",
```

Comment:
nit space before "epoch"

Reply:
that's intentional - if it is a full epoch - it is replaced with empty string. if it is partial epoch , it is replaced with "partial ". no space means we don't end up with double space